openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -154,13 +155,26 @@ def set_logged_in_cookies(request, response, user):
     return response
 
 
-def refresh_jwt_cookies(request, response, user):
+def get_response_with_refreshed_jwt_cookies(request, user):
     """
-    Resets the JWT related cookies in the response for the given user.
+    Generates the response and resets the JWT related cookies in the response for the given user.
     """
     cookie_settings = standard_cookie_settings(request)
+    response = JsonResponse({})
     _create_and_set_jwt_cookies(response, request, cookie_settings, user=user)
 
+    current_time = time.time()
+    expires_date = cookie_settings.get('expires', 'not-found')
+    expires_epoch = 0 if expires_date == 'not-found' else parse_http_date(expires_date)
```

Comment:
Nit: I'd replace `'not-found'` with `None` and just use the `not-found` below when formatting output.